config/config.go

```diff
@@ -1,62 +1,59 @@
 package config
 
 import (
-	"fmt"
 	"io"
 	"io/ioutil"
-	"net/url"
 	"os"
 
 	"github.com/allegro/akubra/log"
+	logconfig "github.com/allegro/akubra/log/config"
 	"github.com/allegro/akubra/metrics"
+	shardingconfig "github.com/allegro/akubra/sharding/config"
 	set "github.com/deckarep/golang-set"
-	"github.com/go-yaml/yaml"
+	"github.com/go-validator/validator"
+	yaml "gopkg.in/yaml.v2"
 )
 
 // YamlConfig contains configuration fields of config file
 type YamlConfig struct {
-	// Listen interface and port e.g. "0:8000", "localhost:9090", ":80"
-	Listen string `yaml:"Listen,omitempty"`
+	// Listen interface and port e.g. "0.0.0.0:8000", "127.0.0.1:9090", ":80"
+	Listen string `yaml:"Listen,omitempty" validate:"regexp=^(([0-9]+[.][0-9]+[.][0-9]+[.][0-9]+)?[:][0-9]+)$"`
+	// List of backend URI's e.g. "http://s3.mydatacenter.org"
+	Backends []shardingconfig.YAMLUrl `yaml:"Backends,omitempty,flow"`
 	// Maximum accepted body size
-	BodyMaxSize string `yaml:"BodyMaxSize,omitempty"`
+	BodyMaxSize string `yaml:"BodyMaxSize,omitempty" validate:"regexp=^([1-9][0-9]+[kMG][B])$"`
```

Comment:
I'm not sure if we should regexp here. Why not use units.FromHumanSize and check if it returns error? We would have covered everything that units.FromHumanSize is able to parse.